### ISSUE TITLE
I have a question in running ProST_example_regi.py

### ISSUE THREAD

**Issue by cyxin-art** [gaocong13/Projective-Spatial-Transformers#31]
I meet an error :
E:\Chormdown\Projective-Spatial-Transformers-master\src\util.py:152: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
rtvec = torch.tensor(rtvec.detach(), requires_grad=True, device=device)

Has the author encountered this problem？

**Comment by gaocong13** [comment]
Hi, sorry for the delayed response. Have you solved this issue? This is a warning message rather than an actual error. You should be able to run through the program without a segfault.

**Comment by cyxin-art** [comment]
Thank you，and I have another question want to ask you. What is the difference from ProST and DRR? Thank you very much and look forward to your reply
